docs/GraFx-Studio/guides/barcodes/overview/index.md

```diff
@@ -0,0 +1,212 @@
+# Overview of Barcodes
```

Comment:
The barcodes on this page are also not in alphabetical order, which makes them harder to find if you are looking for one particular type.

Reply:
I added order in the chaos